easybuild/easyconfigs/n/neko/neko-0.9.1-foss-2025a-cpu.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'neko'
+version = '0.9.1'
+versionsuffix = '-cpu'
+
+toolchain = {'name': 'foss', 'version': '2025a'}
+
+homepage = 'https://neko.cfd/'
+description = """Neko is a portable framework for high-order spectral element flow simulations."""
+
+source_urls = ['https://github.com/ExtremeFLOW/neko/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('json-fortran', '8.3.0'),
```

Comment:
I'd first try it with `json-fortran-9.0.5-GCC-14.2.0.eb` to see if that will work here; otherwise it's introducing a fork where we end up with incompatible modules inside the toolchain which is always a pain in the butt.